R/travis-meta.R

```diff
@@ -1,21 +1,28 @@
-#' Retrieve meta information from Travis CI
+#' @title Retrieve meta information from Travis CI
 #'
 #' @description
 #' Return repositories and user information.
 #'
-#' `travis_repositories()` queries the "/repos" API.
+#' @details
+#' `travis_repositories()` queries the `"/repos"` API.
 #'
 #' @inheritParams travis_repo_info
 #'
-#' @seealso [Travis CI API documentation](https://docs.travis-ci.com/api)
+#' @seealso [Travis CI API documentation](https://developer.travis-ci.com/)
 #'
 #' @family Travis CI functions
 #'
 #' @export
-travis_repos <- function(token = travis_token()) {
-  req <- TRAVIS_GET3("/repos", token = token)
-  httr::stop_for_status(req, paste("list repositories"))
-  new_travis_repos(httr::content(req))
+travis_repos <- function() {
+  req = travisHTTP(path = "/repos")
+
+  if (status_code(req$response) == 200) {
```

Comment:
Do we really need to check the HTTP status code here and everywhere?

Reply:
I guess you mean that if we do it like that, we should also have a "proper" `else`;) Hach, I got a bit tired.

Reply:
We already check in `http.R` for non/20* codes which should be sufficient.

Reply:
I don't mind removing these `if()` conditions everywhere.